core/pom.xml

```diff
@@ -14,7 +14,7 @@
 
     <groupId>ConfigHub</groupId>
     <artifactId>ConfigHub-Core</artifactId>
-    <version>1.8.4</version>
+    <version>1.8.5-alpha-1</version>
```

Comment:
we probably don't want to change the base version master to the alpha.  I get you probably did this to get a build for yourself.